public/css/styles.css

```diff
@@ -120,6 +120,16 @@ td {
 .address + .address {
```

Comment:
You need to add `.address-wrap + .address-wrap` selector, to add spaces between wrapped inputs with addresses.